docs_src/src/pages/documentation/api_reference/getting_started.mdx

```diff
@@ -244,19 +244,20 @@ Batman was curious about how to access path parameters and query parameters from
     from robyn import jsonify
 
     @app.post("/jsonify/:id")
-    async def json(request):
-        print(request.path_params["id"])
+    async def json(path_params):
+        print(path_params["id"])
         return {"hello": "world"}
 
 
     ```
 
     ```python {{ title: 'typed' }}
-    from robyn import jsonify, Request
+    from robyn import jsonify
+    from robyn.types import PathParams
 
     @app.post("/jsonify/:id")
-    async def json(request: Request):
-        print(request.path_params["id"])
+    async def json(pp: PathParams):
```

Comment:
@VishnuSanal , this should be of the same name as the `untyped` version. We should add another example with the description suggestion that you don't need to use the reserved keywords. Also, explaining that the `typing` will take precedence over reserved keywords.